woodwork/table_accessor.py

```diff
@@ -145,6 +146,27 @@ def __getattr__(self, attr):
         else:
             raise AttributeError(f"Woodwork has no attribute '{attr}'")
 
+    def __getitem__(self, key):
+        if isinstance(key, list):
+            index = pd.Index(key)
+            columns = self._dataframe.columns
+            diff = index.difference(columns)
+            in_columns = diff.isin(columns).all()
+
+            if not in_columns:
+                raise KeyError(f"Column(s) '{list(diff)}' not found in DataTable")
+
+            return self._get_subset_df_with_schema(key)
+
+        if key not in self._dataframe:
+            raise KeyError(f"Column with name '{key}' not found in DataTable")
+
+        series = self._dataframe[key]
+        column = deepcopy(self._schema.columns[key])
```

Comment:
If you use the `self.schema` property here instead of `self._schema` we should be able to eliminate the deepcopy and the associated import.

Reply:
✅  479d5a2